BingMaps/v8-web-control/map-control-api/moduleoptions-object.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "ModuleOptions Object | Microsoft Docs"
+description: "This article lists three module options by name, type, and description."
```

Comment:
Let's add API into the description here too.  Maybe something like:
"For the ModuleOptions Object, this article lists three module options by name, type, and description."

or

"This article lists three module options by name, type, and description for the ModuleOptions Object."